.github/workflows/gradle-docker-ci.yml

```diff
@@ -0,0 +1,34 @@
+##
```

Comment:
Could you please bump the `com.gradle.plugin-publish` version in `build.gradle`, and add the `maven-publish` plugin too please

We essentially want to mirror the changes that Ben has done in here: https://github.com/brightsparklabs/gradle-docs/pull/7/files